source/project-stories/plymouth-university.erb

```diff
@@ -0,0 +1,328 @@
+---
+title: "Plymouth University Project Story"
+
+header:
+  client: "Plymouth University"
+  introduction: "Unboxed leads Agile overhaul of Plymouth University website"
+  image: "/project-stories/plymouth-university/plymouth-building.jpg"
+  image_description: "Plymouth university building"
+
+introduction:
+  title: "New website led by Unboxed"
+  content: "With the arrival of expensive tuition fees, universities have had to become more commercially oriented. Plymouth University realised that prospective students learn about them primarily through their website, and that the current site did not reflect their quality. A team from Unboxed worked alongside developers from the University to rapidly overhaul the website, adding functionality in two-week sprints."
+  image: "/project-stories/plymouth-university/plymouth-website.jpg"
+  image_description: "Plymouth website screenshot"
+
+plymouth_cio_video:
+  youtube_id: "nTSKD1PG1e4"
+  thumbnail: "/project-stories/plymouth-university/video-still.jpg"
```

Comment:
Have updated the images in https://github.com/unboxed/ubxd_web_refresh/commit/3e3eced087c945fd738dced2ee00f305701bcf5b. It was referencing the .jpg files but only .png files were present.